### PR TITLE
change language setting by Yansen Liu

### PR DESCRIPTION
**Pull request by liux1731**

### CODE REVIEW

File: project/lang/class.default.php [hunk 1]
```diff
@@ -8,26 +8,25 @@ class DefaultLang
 
 	private $default_js = array
 	(
-		'confirmDelete' => "Are you sure you want to delete the task?",
```
Comment (buse2017fall) [inline project/lang/class.default.php:11]: What's the purpose of your modification ?

File: project/lang/en.php [hunk 2]
```diff
@@ -5,24 +5,23 @@ class Lang extends DefaultLang
 {
 	var $js = array
 	(
-		'confirmDelete' => "Are you sure you want to delete the task?",
+		'confirmDelete' => "Are you sure you want to delete the story?",
```
Comment (buse2017fall) [inline project/lang/en.php:8]: What's the difference of these two files ?

Comment (liux1731) [reply project/lang/en.php:8]: we decide to use "story" rather than "task" in our app

File: project/lang/class.default.php (hunk 1 above)
Comment (liux1731) [reply project/lang/class.default.php:11]: change names of some features and delete some features that we do not need anymore

**Pull request merged**